packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx

```diff
@@ -5,22 +5,32 @@ import './index.scss';
 
 export const PaymentLegalBlurb = () => (
   <div className="payment-legal-blurb">
-    <Localized id="payment-legal-copy">
-      <p>Mozilla uses Stripe for secure payment processing.</p>
+    <Localized id="payment-legal-copy-stripe-paypal">
+      <p>Mozilla uses Stripe and Paypal for secure payment processing.</p>
     </Localized>
 
     <Localized
-      id="payment-legal-link"
+      id="payment-legal-link-stripe-paypal"
       elems={{
-        a: <a
-          href="https://stripe.com/privacy"
-          target="_blank"
-          rel="noopener noreferrer"
-        ></a>
+        stripe: (
+          <a
+            href="https://stripe.com/privacy"
+            target="_blank"
+            rel="noopener noreferrer"
+          ></a>
+        ),
+        paypal: (
+          <a
+            href="https://paypal.com/privacy"
+            target="_blank"
+            rel="noopener noreferrer"
+          ></a>
+        ),
       }}
     >
       <p>
-        View the <a>Stripe privacy policy</a>.
+        View the <stripe>Stripe privacy policy</stripe> and{' '}
```

Comment:
this made react unhappy but i wasn't sure how to ref the elems - guidance appreciated.